README.md

```diff
@@ -75,6 +75,24 @@ Note, you can use any of the supported [storage backends](https://restic.readthe
    ````
 1. (optional) Setup email on failure as described [here](#8-email-notification-on-failure)
 
+# 🏃 Restic wrapper
```

Comment:
Thanks for writing REAME docs ⭐ 

To keep the base install minimal,  simple, easy to follow, let's move this to after "9. Optional: automated backup checks" as "10. Optional: restic profile wrapper" or something like this.

Later in  #58 I will restructure the optionals better in the README

Reply:
Perfect! No worries :)